packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -306,4 +306,298 @@
     </SF.Item>
   </Shw::Flex>
 
+  <Shw::Text::H3>Checkmark</Shw::Text::H3>
+  <Shw::Flex as |SF|>
+    <SF.Item>
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}}>
+            {{state}}
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item>
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}} @selected={{true}}>
+            {{state}}
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item>
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}} @resultCount="12">
+            {{state}}
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item>
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}} @selected={{true}} @resultCount="12">
+            {{state}}
+          </Hds::Dropdown::ListItem::Checkmark>
+        {{/each}}
+      </ul>
+    </SF.Item>
+    <SF.Item>
+      <ul class="hds-dropdown-list" role="listbox">
+        {{#each @model.ITEM_STATES as |state|}}
+          <Hds::Dropdown::ListItem::Checkmark mock-state-value={{state}}>
+            <Shw::Placeholder @text="custom content" @width="150" @height="20" @background="#eee" />
```

Comment:
super-nitpick: if you use `144` instead of `150`, the items will be better aligned (and will make easier for the eye to compare them)

Reply:
Thanks, yes, that was the initial intention, but haven't updated the values after the layout refactor.